tests/integration/profiles_management/conftest.py

```diff
@@ -0,0 +1,34 @@
+import logging
```

Comment:
suggestion: Could you add a docstring describing the utility of the module here?